File: src/main/resources/statics/js/modules/mpwx/mpwxapp.js
$(function () {
    $("#jqGrid").jqGrid({
        url: baseURL + 'mpwx/mpwxapp/list',
        datatype: "json",
        colModel: [			
			{ label: 'id', name: 'id', index: 'id', width: 50, key: true },
			{ label: '公众号名称', name: 'name', index: 'name', width: 80 }, 			
			{ label: '微信原始id', name: 'weixinNo', index: 'weixin_no', width: 80 }, 			
			{ label: '微信appId', name: 'appId', index: 'app_id', width: 80 }, 			
			{ label: '微信secret', name: 'appsecret', index: 'appsecret', width: 80 }, 			
			{ label: 'url', name: 'url', index: 'url', width: 80 }, 			
			{ label: 'token', name: 'token', index: 'token', width: 80 }, 			
			{ label: 'encodingaeskey', name: 'encodingaeskey', index: 'encodingaeskey', width: 80 }, 			
			{ label: '商户号', name: 'partner', index: 'partner', width: 80 }, 			
			{ label: '商户签名', name: 'paysignkey', index: 'paysignkey', width: 80 }, 			
			{ label: '创建时间', name: 'createTime', index: 'create_time', width: 80 }, 			
			{ label: '修改时间', name: 'modifyTime', index: 'modify_time', width: 80 }			
        ],
		viewrecords: true,
        height: 385,
        rowNum: 10,
		rowList : [10,30,50],
        rownumbers: true, 
        rownumWidth: 25, 
        autowidth:true,
        multiselect: true,
        pager: "#jqGridPager",
        jsonReader : {
            root: "page.list",
            page: "page.currPage",
            total: "page.totalPage",
            records: "page.totalCount"
        },
        prmNames : {
            page:"page", 
            rows:"limit", 
            order: "order"
        },
        gridComplete:function(){
        	//隐藏grid底部滚动条
        	$("#jqGrid").closest(".ui-jqgrid-bdiv").css({ "overflow-x" : "hidden" }); 
        }
    });
});

var vm = new Vue({
	el:'#icloudapp',
	data:{
		showList: true,
		title: null,
		mpwxApp: {}
	},
	methods: {
		query: function () {
			vm.reload();
		},
		add: function(){
			vm.showList = false;
			vm.title = "新增";
			vm.mpwxApp = {};
		},
		update: function (event) {
			var id = getSelectedRow();
			if(id == null){
				return ;
			}
			vm.showList = false;
            vm.title = "修改";
            
            vm.getInfo(id)
		},
		saveOrUpdate: function (event) {
		    $('#btnSaveOrUpdate').button('loading').delay(1000).queue(function() {
                var url = vm.mpwxApp.id == null ? "mpwx/mpwxapp/save" : "mpwx/mpwxapp/update";
                $.ajax({
                    type: "POST",
                    url: baseURL + url,
                    contentType: "application/json",
                    data: JSON.stringify(vm.mpwxApp),
                    success: function(r){
                        if(r.code === 0){
                             layer.msg("操作成功", {icon: 1});
                             vm.reload();
                             $('#btnSaveOrUpdate').button('reset');
                             $('#btnSaveOrUpdate').dequeue();
                        }else{
                            layer.alert(r.msg);
                            $('#btnSaveOrUpdate').button('reset');
                            $('#btnSaveOrUpdate').dequeue();
                        }
                    }
                });
			});
		},
		del: function (event) {
			var ids = getSelectedRows();
			if(ids == null){
				return ;
			}
			var lock = false;
            layer.confirm('确定要删除选中的记录？', {
                btn: ['确定','取消'] //按钮
            }, function(){
               if(!lock) {
                    lock = true;
		            $.ajax({
                        type: "POST",
                        url: baseURL + "mpwx/mpwxapp/delete",
                        contentType: "application/json",
                        data: JSON.stringify(ids),
                        success: function(r){
                            if(r.code == 0){
                                layer.msg("操作成功", {icon: 1});
                                $("#jqGrid").trigger("reloadGrid");
                            }else{
                                layer.alert(r.msg);
                            }
                        }
				    });
			    }
             }, function(){
             });
		},
		getInfo: function(id){
			$.get(baseURL + "mpwx/mpwxapp/info/"+id, function(r){
                vm.mpwxApp = r.mpwxApp;
            });
		},
		reload: function (event) {
			vm.showList = true;
			var page = $("#jqGrid").jqGrid('getGridParam','page');
			$("#jqGrid").jqGrid('setGridParam',{ 
                page:page
            }).trigger("reloadGrid");
		}
	}
});

File: src/main/java/com/icloud/modules/mpwx/entity/MpwxApp.java
package com.icloud.modules.mpwx.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

/**
 * 
 * 
 * @author zdh
 * @email yyyyyy@cm.com
 * @date 2020-07-20 15:44:27
 */
@Data
@TableName("t_mpwx_app")
public class MpwxApp implements Serializable {
	private static final long serialVersionUID = 1L;

   	   /*  */
       @TableId(value="id", type= IdType.AUTO)
       private Long id;
   	   	   /* 公众号名称 */
       @TableField("name")
       private String name;
   	   	   /* 微信原始id */
       @TableField("weixin_no")
       private String weixinNo;
   	   	   /* 微信appId */
       @TableField("app_id")
       private String appId;
   	   	   /* 微信secret */
       @TableField("appsecret")
       private String appsecret;
   	   	   /* url */
       @TableField("url")
       private String url;
   	   	   /* token */
       @TableField("token")
       private String token;
   	   	   /* encodingaeskey */
       @TableField("encodingaeskey")
       private String encodingaeskey;
   	   	   /* 商户号 */
       @TableField("partner")
       private String partner;
   	   	   /* 商户签名 */
       @TableField("paysignkey")
       private String paysignkey;
   	   	   /* 创建时间 */
       @TableField("create_time")
       private Date createTime;
   	   	   /* 修改时间 */
       @TableField("modify_time")
       private Date modifyTime;
   	
}
